scripts/docgen-compat/theme/layouts/default.hbs

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="hide_page_heading" value="true" />
-    <meta name="project_path" value="/docs/reference/js/_project.yaml" />
+    <meta name="project_path" value="/docs/reference/node/_project.yaml" />
```

Comment:
It doesn't matter what this "value" is set to before the script runs, the docgen script will swap it to the correct one when it runs.